### PR TITLE
Library upgrades mid 2020

### PR DESCRIPTION
**Pull request by rcillo**
Upgrading libraries impacted by CVEs.

Most of the security issues wouldn't be exploitable but just to stay on the safe side, we are upgrading all of those which have known security issues.

The current state of dependency management in this project requires some improvement. There's a lot of duplication and editing things often means editing multiple places. But I would leave a reorganization to a different task in order to keep the focus on the upgrades.

### CODE REVIEW

**Comment by antban** [comment]
:+1:

File: app/build.gradle [hunk 1]
```diff
@@ -115,13 +117,16 @@ dependencies {
     compile 'org.apache.commons:commons-lang3:3.8.1'
     compile 'org.zalando:nakadi-plugin-api:3.2.1'
     compile 'org.echocat.jomon:runtime:1.6.3'
+    compile 'com.googlecode.libphonenumber:libphonenumber:8.12.7'
```
[the remark this answers is not in the document]
Comment (antban) [reply app/build.gradle:119]: Found why (echocat)

[the remark this answers is not in the document]
Comment (rcillo) [reply app/build.gradle:119]: I kept a copy of all the CVEs but they were so many that I didn't read all.

**Comment by antban** [comment]
:+1:

**Comment by rcillo** [comment]
👍

**Pull request merged**